### ISSUE TITLE
College Success: Suppress Answer Key

### ISSUE THREAD

**Issue by openstaxalina** [openstax/cnx-recipes#1647]
@clteam18 commented on [Fri Nov 22 2019](https://github.com/openstax/oer.exports/issues/3988)

- **Book(s)**: College Success
- **URL/Server**: PDF
- **Location (page number, chapter/section)**: Chapter 03/ PDF page 49
- **Suggested nearby searchable text**: Answer Key

# Screenshots

![image](https://user-images.githubusercontent.com/39794953/69426354-1ab90080-0d53-11ea-8fbf-8a6cf1a2cabe.png)

# Additional notes

Unknown page is created. Please advise.

---

@openstaxalina commented on [Fri Nov 22 2019](https://github.com/openstax/oer.exports/issues/3988#issuecomment-557662967)

I believe this book will not have an Answer Key, and thus the Answer Key needs to be suppressed/removed.  @yblock @lwtchu is this also your understanding?  If so will confirm with Anthony.

---

@yblock commented on [Fri Nov 22 2019](https://github.com/openstax/oer.exports/issues/3988#issuecomment-557679095)

I believe you are correct.

---

@openstaxalina commented on [Tue Nov 26 2019](https://github.com/openstax/oer.exports/issues/3988#issuecomment-558752171)

Anthony confirmed - no Answer Key in this title.

**Comment by openstaxalina** [comment]
Test content: 
https://easyvm5.cnx.org/contents/h2TUSN8V@3.1:Ig9JNInZ@2/Introduction
(long UUID: 8764d448-df15-4479-9df6-c93ab5c6176e on easyvm5)

**Comment by brittany-johnson** [comment]
Assigning @Stackblocks to use this issue for recipes training cc: @helenemccarron

**Comment by brittany-johnson** [comment]
Update: There currently isn't an answer key in the recipe

**Comment by omehes** [comment]
I used critical.cher branch to generate a pdf of COSU book (col29727). No Answer Key is present. 
fixed
@brittany-johnson 

[col29727 (4).pdf](https://github.com/openstax/cnx-recipes/files/4123186/col29727.4.pdf)

**Comment by openstaxalina** [comment]
@omehes this issue relates to the recipe (the webview) of the book.  Could you confirm you do not see the Answer Key in the webview?

cc @brittany-johnson @Stackblocks in case I mis-stated anything.

**Comment by yblock** [comment]
https://easyvm5.cnx.org/contents/h2TUSN8V@2.1:Ig9JNInZ@2/Introduction

Here is a version with the recipe applied and collated sections. The Answer Key was not displayed in web, only in the PDF. :)

**Comment by omehes** [comment]
@openstaxalina @yblock does the above message mean that my test is valid (no Answer key in pdf)?

**Comment by openstaxalina** [comment]
@omehes we can keep this issue closed.  This is what I understand about where/how to test various issues:

- Issues in `oer.exports` repo: test with PDF generated on the server (not with CLI, not with COPS)
- Issues in `cnx-recipes` repo without the `baked.styles` label: test on the webview on the server
- Issues in `cnx-recipes` repo with the `baked.styles` label: test with PDF generated by CLI or COPS

Here is what I think happened with this specific issue:

1. Production team opened issue in `oer.exports` because of Answer Key in PDF: https://github.com/openstax/oer.exports/issues/3988
2. We removed the Answer Key in the PDF.
3. I incorrectly assumed that the Answer Key would also need to be removed in the recipe/webview, so I opened this issue in `cnx-recipes`.
4. The Answer Key never existed in the webview, so there was no coding work for this card.

Sorry about that!  😩
